docs/source/azure/configuration/acitutorial.rst

```diff
@@ -82,16 +98,25 @@ Create a parameters file called ``c7n-aci-tutorial.json``
                 "value": "c7n-aci"
             },
             "azure_subscription_id": {
-                "value": "<Target Subscription Id>"
+                "value": "<Target Subscription ID>"
             },
-            "azure_event_queue_name": {
+            "azure_container_queue_name": {
```

Comment:
nit: would be great to somehow separate c7n Container Host & Azure Storage Container.. Feel a little confusing

Reply:
Do you mean semantically separate them? I agree that it is a bit confusing. I could specify "blob container" vs "container host".